src/vm/readytoruninfo.cpp

```diff
@@ -614,6 +615,21 @@ ReadyToRunInfo::ReadyToRunInfo(Module * pModule, PEImageLayout * pLayout, READYT
         }
     }
 
+    // For format version 3.1 and later, there is an optional attributes section
+    if (IsImageVersionAtLeast(3,1))
```

Comment:
Nit: I know that you are following the existing pattern, but these `IsImageVersionAtLeast` checks are unnecessary. (I think I have made the same comment the last time new structure got added here.)